image/image.go

```diff
@@ -63,7 +63,7 @@ type localInfos struct {
 
 func (li *localInfos) Name(pathOrName string) string {
```

Comment:
we want a TODO in this file about considering to which extent allowing parallel installs when building images

Reply:
>  we want a TODO in this file about considering to which extent allowing parallel installs when building images

We should probably block it completely for now, and lifting the restriction once everything else works and we are more confident of potential issues.